<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTipDuration">
    <number>-3</number>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>240</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>ForbiddenCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#666666;
font-family:Bebas Neue;
font-size:17px;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;QCMessanger&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="input_mes">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>611</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QTextEdit {
	background-color:#ffffff;
	border-radius:25px;
	border:2px solid #dcdcdc;
	color:#000000;
	font-family:Bebas Neue;
	font-size:17px;
}

</string>
    </property>
    <property name="placeholderText">
     <string>Enter your message here</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sen_mes">
    <property name="geometry">
     <rect>
      <x>639</x>
      <y>459</y>
      <width>141</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QPushButton {
	background-color:#ffffff;
	border-radius:15px;
	border:2px solid #dcdcdc;
	color:#666666;
	font-family:Arial;
	font-size:15px;
	font-weight:bold;
}
.QPushButton:hover {
	background-color:#f6f6f6;
	color:#121212;
}
.QPushButton:pressed {
	border-color:#121212;
}</string>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>618</x>
      <y>548</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#121212;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Status: |not sent|&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="music">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>360</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">.QPushButton {
	background-color:#ffffff;
	border-radius:25px;
	border:2px solid #dcdcdc;
	color:#666666;
	font-family:Arial;
	font-size:15px;
	font-weight:bold;
}
.QPushButton:hover {
	background-color:#f6f6f6;
	color:#121212;
}
.QPushButton:pressed {
	border-color:#121212;
}</string>
    </property>
    <property name="text">
     <string>Music</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="covid">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>110</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QPushButton {
	background-color:#ffffff;
	border-radius:25px;
	border:2px solid #dcdcdc;
	color:#666666;
	font-family:Arial;
	font-size:15px;
	font-weight:bold;
}
.QPushButton:hover {
	background-color:#f6f6f6;
	color:#121212;
}
.QPushButton:pressed {
	border-color:#121212;
}</string>
    </property>
    <property name="text">
     <string>Covid-19
 Info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="weather">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>230</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QPushButton {
	background-color:#ffffff;
	border-radius:25px;
	border:2px solid #dcdcdc;
	color:#666666;
	font-family:Arial;
	font-size:15px;
	font-weight:bold;
}
.QPushButton:hover {
	background-color:#f6f6f6;
	color:#121212;
}
.QPushButton:pressed {
	border-color:#121212;
}</string>
    </property>
    <property name="text">
     <string>Weather</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>ForbiddenCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#666666;
font-family:Bebas Neue;
font-size:17px;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Private dialog with:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_message_to">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>ForbiddenCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#666666;
font-family:Bebas Neue;
font-size:17px;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="pprint_mes">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>641</width>
      <height>331</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"> .QTextBrowser{
	background-color:#ffffff;
	border-radius:25px;
	border:2px solid #dcdcdc;
	color:#000000;
	font-family:Bebas Neue;
	font-size:17px;
}
</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Bebas Neue'; font-size:17px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
